xmodule/util/builtin_assets.py

```diff
@@ -29,7 +29,8 @@ def add_css_to_fragment(fragment, css_relative_path):
     css_absolute_path = Path(settings.REPO_ROOT) / "xmodule" / "assets" / css_relative_path
     if not css_absolute_path.is_file():
         raise FileNotFoundError(f"css file not found: {css_absolute_path}")
-    fragment.add_css_url(str(css_absolute_path))
+    # fragment.add_css_url(str(css_absolute_path))
```

Comment:
@kdmccormick I have changed css url addition to css content addition as css url is not accessible on browser and css file needs to add in the static files.

Please guide what should be the way forward of implementation?